worker-markup-core/src/test/java/com/github/cafdataprocessing/worker/markup/core/MarkupOfHeadersAndBodyTest.java

```diff
@@ -110,11 +110,8 @@ public void testCondensedHeaderMultiLangMappings() throws IOException, JDOMExcep
     public void testFalsePositiveCondensedHeaderMultiLangMappings() throws IOException, JDOMException
     {
         final Document xmlDocument = TestUtility.readXmlFile("src/test/resources/xml/FalsePositiveCondensedHeaderMultiLangMapping.xml");
-        try {
-            MarkupHeadersAndBody.markUpHeadersAndBody(xmlDocument, emailHeaderMappings, condensedHeaderMultiLangMappings);
-        } catch (final RuntimeException ex) {
-            assertTrue(ex.getMessage().equals("Logic error detected.  E-mail fidelity has been lost!"));
-        }
+        MarkupHeadersAndBody.markUpHeadersAndBody(xmlDocument, emailHeaderMappings, condensedHeaderMultiLangMappings);
+        //No assert nessesary, if this test had failed it would have thrown a RuntimeException.
```

Comment:
The other tests show the output.  This one does not.

Reply:
The other tests verify the marking up of certain sections of a header, this test is to verify no exception is thrown which is why it's not required.